src/main/java/net/sf/jabref/gui/FieldSetComponent.java

```diff
@@ -149,13 +156,23 @@ private FieldSetComponent(String title, List<String> fields, List<String> preset
         con.weightx = 1;
         if (preset == null) {
             input = new JTextField(20);
-            input.addActionListener(this);
+            input.addActionListener(e -> addField(input.getText()));
             gbl.setConstraints(input, con);
             add(input);
         } else {
             sel = new JComboBox<>(preset.toArray(new String[preset.size()]));
             sel.setEditable(true);
-            //sel.addActionListener(this);
```

Comment:
Usually, commented-out code should be deleted. So instead of extending them, please remove these lines.